### ISSUE TITLE
Does the changelog generator handle a commit closing multiple tickets correctly?

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5-dev#70]
I didn't have time to check, so this is a TODO. Validate that the following merge commit will be handled correctly:

> A bug fix for an existing feature (closes two tickets):
>
> ```
>Fix: The editor will be great again. Closes: #3. Closes: #4.
> ```

BTW. The repeated `Closes` is proposed because, AFAIR (but it might be checked too), listing issues like `Closes: #3, #4.` doesn't work – GitHub would not close all the mentioned tickets.

**Comment by pomek** [comment]
This is a question to GitHub :)

But I'll check it.

**Comment by pomek** [comment]
```
[…] Closes #n. Closes #n+1.
```

It will work (https://github.com/Reinmar/ckeditor5-a/issues/1, https://github.com/Reinmar/ckeditor5-a/issues/2).

**Comment by pomek** [comment]
```
[…] Closes #n, #n+1
```

It won't work (https://github.com/Reinmar/ckeditor5-a/issues/3, https://github.com/Reinmar/ckeditor5-a/issues/4).

**Comment by Reinmar** [comment]
OK, thanks.

**Comment by pomek** [comment]
```
Feature: Main module displays `console.log`. Closes #5.

It resolves #6.

NOTE: It should also close #4.

BREAKING CHANGE: The main module will display `console.log`.
```

It closes all tickets described in commit.

**Comment by pomek** [comment]
Generated changelog looks nice.

---

Changelog
=========

## [1.0.0](https://github.com/Reinmar/ckeditor5-a/compare/v0.4.5...v1.0.0) (2017-02-06)

### Features

* Added gulp and changelog task. Closes [#1](https://github.com/Reinmar/ckeditor5-a/issues/1). Closes [#2](https://github.com/Reinmar/ckeditor5-a/issues/2). ([ec57e76](https://github.com/Reinmar/ckeditor5-a/commit/ec57e76))
* Main module displays `console.log`. Closes [#5](https://github.com/Reinmar/ckeditor5-a/issues/5). ([334a492](https://github.com/Reinmar/ckeditor5-a/commit/334a492)), closes [#6](https://github.com/Reinmar/ckeditor5-a/issues/6)
* Removed empty file. Closes [#3](https://github.com/Reinmar/ckeditor5-a/issues/3), [#4](https://github.com/Reinmar/ckeditor5-a/issues/4). ([8a4b24d](https://github.com/Reinmar/ckeditor5-a/commit/8a4b24d))


### BREAKING CHANGE

* The main module will display `console.log`.

### NOTE

* It should also close [#4](https://github.com/Reinmar/ckeditor5-a/issues/4).

---

> Main module displays console.log. Closes #5. (334a492), closes #6

Because the commit message was invalid, generated entry in changelog is also invalid.

**Comment by pomek** [comment]
> Main module displays console.log. Closes #5. (334a492), closes #6

Hmm… If we put in the commit message ticket IDs, they will be put in the changelog entry.

**Comment by pomek** [comment]
For the commit:

```
 Fix: Message commit.

 Message body.

 * Resolve: #1.
 * Resolves #2.
 * Close: #3.
 * Closes: #4.
 * Close #5.
 * Close #6.

 NOTE: Nothing to add.
```

generated changelog will look:

---

Changelog
=========

## [1.0.1](https://github.com/Reinmar/ckeditor5-a/compare/v1.0.0...v1.0.1) (2017-02-06)

### Bug fixes

* Message commit. ([c086a06](https://github.com/Reinmar/ckeditor5-a/commit/c086a06)), closes [#2](https://github.com/Reinmar/ckeditor5-a/issues/2) [#5](https://github.com/Reinmar/ckeditor5-a/issues/5) [#6](https://github.com/Reinmar/ckeditor5-a/issues/6)


### NOTE

* Nothing to add.
